w3-blob.md

```diff
@@ -150,8 +155,28 @@ Shows an example receipt for the above `/space/content/add/blob` capability invo
           }
         }
       },
-      // 2. System will attempt to accept received content
-      // if matches blob multihash and size.
+      // 2. System requests user agent (or anyone really) to upload the content
+      // corresponding to the blob
+      // via HTTP PUT to given location.
+      { // "/": "bafy...upload",
+        "cmd": "/http/put",
+        "sub": "did:key:zMh...der", // <-- Ed299.. derived key from content multihash
+        "args": {
+          "content": { "/": { "bytes": "mEi...sfKg" } },
+          "address": {        
+            "url": "https://r2.cloudflare.com/ipfs/bafy...",
+            "headers": {}
+          }
+        },
+        "meta": {
+          // archive of the principal keys
+          "keys": {
+            "did:key:zMh...der": { "/": "mEi...sfKg" } 
+          }
+        }
+      },
```

Comment:
while I understand and like the idea of using this, I am actually now failing to understand how this pans out in practise. If we provide effect with CID for this task on the receipt of `blob/add`, how can client receiving this receipt know the key to invoke `http/put`? 

So, I am only seeing that this should go on the receipt of `blob/add` as well, or client would need to find this task content stored somewhere (we currently only store task when it is received to be executed...)